assets/src/edit-story/components/helpCenter/quickTip/index.js

```diff
@@ -68,11 +69,23 @@ const Paragraph = styled(Text)`
   }
 `;
 
+const DoneContainer = styled.div`
+  ${themeHelpers.centerContent}
+  height: 180px;
+  margin-bottom: ${GUTTER_WIDTH}px;
+  svg {
+    width: 135px;
+    height: 135px;
+    display: block;
+  }
```

Comment:
Could we just do `<DoneCheckmark width={135} height={135} />` below? Does that work?

Reply:
it does! thank you